src/py/aspen/database/models/entity.py

```diff
@@ -123,3 +123,29 @@ def get_children(
                 results.append(tup)
 
         return results
+
+    def accessions(self) -> Sequence[Accession]:
+        from .accessions import Accession
```

Comment:
why add this import in the methods like this?

Reply:
Accessions imports Entity, so if I import Accessions from Entity, it'll be a recursive import.

Reply:
do you think there's a way to reorganize the files so we don't have to do this? I find these workarounds for circular dependencies a little confusing.. if not maybe we can add this to our agenda to resolve later on?

Reply:
Not really. :(